src/rollup/types.d.ts

```diff
@@ -1,9 +1,15 @@
 export const VERSION: string;
 
-type FalsyValue = false | null | undefined;
+// utils
+type NullValue = null | undefined | void;
+type FalsyValue = false | NullValue;
```

Comment:
Thinking about this one, I find `FalsyValue` slightly misleading as there exist more kinds of falsy values than are contained in this type, i.e. `0` and `""`. Maybe it is better to remove it and spell out `NullValue | false` in the (only) place where it is used for clarity?